### ISSUE TITLE
[Poll] Poll shared from spaceA isn't displayed for spaceB's members whom are not already member of spaceA

### ISSUE THREAD

**Issue by Julien-Dubois-eXo** [Meeds-io/meeds#467]
**Is your feature request related to a problem? Please describe.**
if a user shares a poll from space A to space B, members of space B who are not members of space A, see only the text appended to the short message or empty publication if the activity contains only the poll

**Describe the solution you'd like**
Here is the proposition of the solution done by the customer: Display content of the shared poll and permit users where that poll was shared to vote.

**Describe alternatives you've considered**
Facebook also has a poll and group feature an evaluation of how they deal with the permission to vote between groups could be evaluated when the need will be worked. 

**Additional context**
We need just to validate that the problem is not expected and a solution should be implemented to address this point.

**Comment by Julien-Dubois-eXo** [comment]
@Samuel-Renault-eXo c'est une demande de PFR que nous avons qui vous impact également. Je souhaiterai avoir ton évaluation également avant de donner une réponse. 

Je suis ok pour accepter le besoin.

**Comment by srenault-meeds** [comment]
Hello @Julien-Dubois-eXo,

> Display content of shared poll and permit users where that poll was shared to vote.

Not sure this is expected actually. How would you deal with the different votes in different spaces? 
I'd rather go for a read-only view of the poll to make sure we don't mix results.
And by hovering the poll object in the shared activity, I should have an information telling me:
- If members of the original space: "Click to access the initial activity and to contribute to the poll"
- If not members of the original space: "This poll is shared to you only for reading purpose."

WDYT?

**Comment by Julien-Dubois-eXo** [comment]
@Samuel-Renault-eXo I have updated the description as discussed.

**Comment by Julien-Dubois-eXo** [comment]
Task related on eXo board: https://community.exoplatform.com/portal/dw/tasks/taskDetail/61333

**Comment by srenault-meeds** [comment]
> @Samuel-Renault-eXo I have updated the description as discussed.

Thanks!

**Comment by Julien-Dubois-eXo** [comment]
@srenault-meeds any update please? Are you ok to accept this need?

**Comment by srenault-meeds** [comment]
Yes indeed I have already put it on identified status. It remains to study a solution .
